src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.star__line {
+  display: flex;
+}
+
+.star__line :last-child {
+  margin-right: 0;
+}
+
+.srar__style {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
The class name '.srar__style' contains a typo. It should be '.star__style' to match the intended class name used in the HTML file and ensure the styles are applied correctly.